### DISCUSSION TITLE
How to import a theme's `.json` file?

### DISCUSSION THREAD

**Discussion by beachwood23** [labithiotis/zed-themes#32] [Q&A] [answered]
First, this tool is sweet. Thanks for making it. I would like to import an edited theme into Zed. 

I am new to Zed. I don't see any documentation on how to do this, or where to place the theme's `.json` file once it is downloaded. Other Zed config settings are in `~/.config/zed`, but the themes do not seem to live here.

Where can this be done, and is it possible to add this to the documentation, if it is missing?

Gracias 🙏

**Comment by 2zqa** [comment, marked as answer]
I managed to install themes by moving it to the `~/.config/zed/themes/` folder. It already existed for me.

That said, I agree this should definitely be added to the documentation.

**Reply by beachwood23** [reply]
This worked, thank you.

**Comment by labithiotis** [comment]
@2zqa thanks for replying quickly, I will also see about exposing documentation in README.md and adding a documentation page

**Comment by beachwood23** [comment]
Made a quick addition to the README here: https://github.com/labithiotis/zed-themes/pull/33
